gwt-timer/pom.xml

```diff
@@ -1,30 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!--
-
-    Copyright © 2019 The GWT Authors
-
-    Licensed under the Apache License, Version 2.0 (the "License");
-    you may not use this file except in compliance with the License.
-    You may obtain a copy of the License at
-
-        http://www.apache.org/licenses/LICENSE-2.0
-
-    Unless required by applicable law or agreed to in writing, software
-    distributed under the License is distributed on an "AS IS" BASIS,
-    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-    See the License for the specific language governing permissions and
-    limitations under the License.
-
--->
 <project xmlns="http://maven.apache.org/POM/4.0.0"
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
-  <groupId>org.gwtproject.timer</groupId>
+  <parent>
```

Comment:
Confirming, we want an explicit parent here? this means that the parent will have to be deployed to central, etc.

Reply:
No, we will not deploy the parent pom. This is Maven Magic (flatten-m-p) and pretty cool.

We have the following pom:
```xml
  <parent>
    <groupId>org.gwtproject.timer</groupId>
    <artifactId>gwt-timer-parent</artifactId>
    <version>dev</version>
  </parent>
  <artifactId>gwt-timer</artifactId>
  <packaging>gwt-lib</packaging>
  <version>${revision}</version>
```
and the flatten-m-p change it to:
```xml
    <modelVersion>4.0.0</modelVersion>
  <groupId>org.gwtproject.timer</groupId>
  <artifactId>gwt-timer</artifactId>
  <version>HEAD-SNAPSHOT</version>
  <packaging>gwt-lib</packaging>
```
 Just build the project locally and check `.flattened-pom.xml`.

Reply:
Ah, i missed that part, thank you!